source/grid/grid_generator.cc

```diff
@@ -7093,8 +7093,9 @@ namespace GridGenerator
 
   template <int dim, int spacedim>
   void
-  convert_hypercube_to_simplex_mesh(const Triangulation<dim, spacedim> &in_tria,
-                                    Triangulation<dim, spacedim> &out_tria)
+  convert_hypercube_to_simplex_mesh_develop(
```

Comment:
I don't think you wanted to rename the function, did you?

Reply:
of course not